### ISSUE TITLE
Airsonos sound delay

### ISSUE THREAD

**Issue by ruyt** [stephen/airsonos#263]
hello Stephen,
I downloaded the latest version of airsonos yesterday. Streaming from my new Apple tv 4 there is a Huge sound delay. Do you know the solution for this problem?

Regards,
Ruyt

**Comment by calumbrodie** [comment]
There is no solution for this problem - meaning this is not a code/software issue.

You should use an application like VLC (via a mac proper), or Kodi (via Mac/Pc or Amazon Fire) which is able to introduce a delay to the video to compensate for the audio delay.

**Comment by alexwall** [comment]
Which part of airsonos is responsible for this delay. 
As I understand airsonos re-encodes the audio stream and send it to the sonos system. 
I use a raspberry pi 2. Would it be better with raspberry pi 3 or a even more powerful hardware?

**Comment by anderskitson** [comment]
Hmm. Yeah I get this delay on my MacBook, so I don't know if it's a hardware issue, But I really don't know. I wonder if for chrome for example could introduce a sound delay in a extension.

**Comment by petergeneric** [comment]
I think this is a duplicate of issue https://github.com/stephen/airsonos/issues/44 which depends on https://github.com/stephen/nodetunes/issues/40

**Comment by stef-germany** [comment]
Synchronisation seems to be the most wanted feature and at the same time difficult to implement. I will pay EUR 300 to the person who finds a solution and makes it available to everybody here at github. Anybody else willing to contribute money to this project?

**Comment by ovk23** [comment]
@stef-germany , yes certainly I am willing to, lets see. I am working on this issue by myself too..

**Comment by rkpatel33** [comment]
I would chip in for this as well.
